PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,18 @@
 ## Description
 
-*(Instructions: replace text in this and all sections with your own text)*
+Provide a detailed description of what this PR does. What problem does it fix? What new capability does it add?
 
-Provide a detailed description of what this PR does.
+## Issue(s) addressed
 
-What problem does it fix? What new capability does it add?
-
-[See the JEDI Documentation](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/pullrequest.html) for further information.
-
-### Issue(s) addressed
-
-Link the issues to be closed with this PR
-- fixes #<issue_number>
-
-If this does not close an existing issue, then [be sure to add a Pipeline, Label, Estimate, Assignees, and Epic](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/issues.html)
-
-## Acceptance Criteria (Definition of Done)
-
-What does it mean for this PR to be finished?
+Fixes #<issue_number>
```

Comment:
W could we even maybe remove this section? Github has another place where you can link an associated issue. But it's not as obvious. But I do find that I end up entering the information twice.

Reply:
If you use the correct syntax you automatically get the gihub r.h.s populated, and doing it this way is easier to find issues across repositories. I think it's better to keep it

Reply:
I am firmly in support of using "fixes" (or any other keyword with the same functionality) because it automatically closes issues after merging PRs, and I find that very helpful. At least last time I checked the linking of an associated issue did the link but didn't close the issue when the PR was merged. Using "fixes" also creates links in associated issues, so I think it's enough to just use "fixes" (without extra step of linking an issue).

Reply:
Oh I didn't know that "fixes" was a keyword! I always changed the wording for non-bug PR's because I felt "fixes" implied a bug. Maybe that's why it didn't work for me!

Reply:
Some info [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests). We could change it to "closes" or "resolves", they have the same behavior

Reply:
Personally I like "resolves". For what it's worth.

Reply:
I've changed to "resolves".